specification/app/Microsoft.App/models/code-execution.tsp

```diff
@@ -36,25 +36,25 @@ union CodeInputType {
   string,
 
   @doc("Inline.")
-  inline: "inline",
+  inline: "Inline",
 
   @doc("InlineBase64.")
-  inlineBase64: "inlineBase64",
+  inlineBase64: "InlineBase64",
 
   @doc("InlineText.")
-  inlineText: "inlineText",
+  inlineText: "InlineText",
 }
 
 @doc("Execution Type")
 union ExecutionType {
   string,
 
   @doc("Synchronous")
-  synchronous: "synchronous",
+  synchronous: "Synchronous",
 }
 
 @doc("The properties of the code execution result.")
-model CodeExecutionResponseProperties {
+model SessionCodeExecutionResponseProperties {
```

Comment:
We don't need to request param under properties as it can be top level properties too. Properties bag concept is applicable for ARM control-plane APIs not really applicable for data-plane APIs I mean have this flat like 

`{ "codeInputType": "Inline", "executionType": "Synchronous", "code": "string", "timeoutInSeconds": 0 } `